conversational-client/src/app/components/history-tutor/history-tutor.component.ts

```diff
@@ -0,0 +1,17 @@
+import {Component} from '@angular/core';
+
+@Component({
+  selector: 'app-history-tutor',
+  templateUrl: './history-tutor.component.html',
+  styleUrl: './history-tutor.component.css',
+})
+export class HistoryTutorComponent {
+  worldState: object | undefined = undefined;
+  worldStateDisplay: string | undefined = undefined;
+
+  updateWorldState(worldState: object): object {
```

Comment:
do we need to keep track of the world state on each component? we're already saving it in app.component, so we could bind a display property in these components, wdyt?

Reply:
you mean just get rid of the worldstate member in the component? 
i think that'll work find for these examples at least so will do that.
let me know if there's a cleverer thing that you mean.

out of interest -- I guess these are just references, so is there any memory issue having the local reference, or is it just a neatness thing? (which is also perfectly valid).

Reply:
I think we'd be able to get rid of it the component, declare it as a property, and bind it to the referenc in app.component. for me it's the latter, I think keeping data and component interactions as clean as possible helps with code health